x/stakeibc/keeper/msg_server_claim_undelegated_tokens.go

```diff
@@ -16,6 +16,28 @@ import (
 func (k msgServer) ClaimUndelegatedTokens(goCtx context.Context, msg *types.MsgClaimUndelegatedTokens) (*types.MsgClaimUndelegatedTokensResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
+	userRedemptionRecord, err := k.GetClaimableRedemptionRecord(ctx, msg)
+	if err != nil {
+		return nil, sdkerrors.Wrapf(err, "unable to find claimable redemption record")
+	}
+
+	icaTx, err := k.GetRedemptionTransferMsg(ctx, userRedemptionRecord, msg.HostZoneId)
+	if err != nil {
+		return nil, sdkerrors.Wrapf(err, "unable to build redemption transfer message")
+	}
+
+	sequence, err := k.SubmitTxs(ctx, icaTx)
```

Comment:
```suggestion
	sequence, err := k.SubmitTxs(ctx, icaTx.ConnectionId, icaTx.Messages, icaTx.Account, icaTx.Timeout)
```
In the interest of changing as few lines of code as possible, wdyt of leaving the SubmitTxs function signature unchanged?

Reply:
yeah good call!